src/CHANGES.txt

```diff
@@ -17,6 +18,9 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
     - Drop support for Python 2.7. SCons will be Python 3.5+ going forward.
 
 
+  From Adam Gross:
+    - Added new module SCons.Scanner.Python to allow scanning .py files.
```

Comment:
Add something like "the python tool is not added by default. Please use env.Tool('python') to add this."